Content.Client/Weapons/Melee/MeleeWeaponSystem.Effects.cs

```diff
@@ -41,6 +42,19 @@ public override void DoLunge(EntityUid user, Angle angle, Vector2 localPos, stri
             return;
         }
 
+        var spriteRotation = Angle.Zero;
+        if (arcComponent.Animation != WeaponArcAnimation.None
+            && TryComp<MeleeWeaponComponent>(weapon, out var meleeWeaponComponent))
+        {
+            if (meleeWeaponComponent.UseWeaponAsAttackAnimation
+                && TryComp<SpriteComponent>(weapon, out var weaponSpriteComponent))
+                sprite.CopyFrom(weaponSpriteComponent);
```

Comment:
Please use the other versions of TryComp

Reply:
did